config/workflowGraph.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const Joi = require('joi');
+const { jobname, requiresValue } = require('../config/job');
+
+const SCHEMA_WORKFLOW_GRAPH = Joi.object().keys({
+    nodes: Joi.array().items(Joi.object().keys({
+        name: requiresValue
```

Comment:
Node and edge objects may be too specific. Does this truncate keys that aren't listed? I'm certain we will want to add more data in here later.

Reply:
Yes, this will ignore keys that are not listed. At first, I was thinking if we want to add some other fields, we modify this data-schema to add it. But if we one this to be done once, I can make it just as an object.

Reply:
I agree with @minz1027. I like being very explicit with what we expect here so there's no magic or confusion.

Reply:
@petey Makes sense to allow us try out cool stuff without modifying data-schema every time. Changed it to allow unknown fields. https://github.com/hapijs/joi/blob/v11.3.4/API.md#objectunknownallow